aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py

```diff
@@ -20,47 +20,19 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import javalang
-
 from aibolit.utils.ast_builder import build_ast
+from aibolit.utils.ast import AST, ASTNodeType
 
 
 class ForceTypeCastingFinder:
 
-    def __process_node(self, node):
-        line = node.position.line if hasattr(node, 'position') and node.position is not None else None
-        qualifier = node.qualifier if hasattr(node, 'qualifier') else None
-        member = node.member if hasattr(node, 'member') else None
-        name = node.name if hasattr(node, 'name') else None
-        return {
-            "line": line,
-            "name": qualifier or member or name,
-            "ntype": type(node)
-        }
-
-    def __tree_to_list(self, tree: javalang.tree.CompilationUnit):
-        '''Convert AST tree to list of object'''
-        items = [self.__process_node(node) for path, node in tree if node is not None]
-
-        # fill missed line numbers
-        last_line_number = None
-
-        for item in items:
-            if (item['line']) is not None:
-                last_line_number = item['line']
-                continue
-            item['line'] = last_line_number
-
-        return items
-
     def value(self, filename: str):
         ''''''
-        tree = build_ast(filename)
-        list_tree = self.__tree_to_list(tree)
-        num_str = []
-        for node in list_tree:
-            if node['ntype'] == javalang.tree.Cast:
-                k = int(node['line'])
-                if k not in num_str:
-                    num_str.append(k)
-        return num_str
+        tree = AST(build_ast(filename))
+        lines = []
```

Comment:
Empty list needs to be specified with types
```suggestion
        lines: List[int] = []
```

Reply:
ok